### PR TITLE
fix(ELB): fix elb active standby pool resource

### PR DESCRIPTION
**Pull request by houpeng80**
**What this PR does / why we need it**:
   fix elb active standby pool resource
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
   fix elb active standby pool resource
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST=./huaweicloud/services/acceptance/elb/ TESTARGS='-run TestAccElbActiveStandbyPool_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/elb/ -v -run TestAccElbActiveStandbyPool_basic -timeout 360m -parallel 4
=== RUN   TestAccElbActiveStandbyPool_basic
=== PAUSE TestAccElbActiveStandbyPool_basic
=== CONT  TestAccElbActiveStandbyPool_basic
--- PASS: TestAccElbActiveStandbyPool_basic (52.47s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/elb       52.514s
```

### CODE REVIEW

File: docs/resources/elb_active_standby_pool.md [hunk 1]
```diff
@@ -64,7 +63,7 @@ The following arguments are supported:
 * `healthmonitor` - (Required, List, ForceNew) Specifies the health check configured for the active/standby backend
   server group. The [healthmonitor](#ELB_healthmonitor) structure is documented below.
 
-* `type` - (Optional, String) Specifies the type of the backend server group. Value options:
+* `type` - (Optional, String, ForceNew) Specifies the type of the backend server group. Value options:
```
Comment (Jason-Zhang9309) [inline docs/resources/elb_active_standby_pool.md]: missing "Changing this ..."
please check other ForceNew parameters

File: docs/resources/elb_active_standby_pool.md [hunk 2]
```diff
@@ -92,37 +90,36 @@ The following arguments are supported:
...
 * `address` - (Required, String, ForceNew) Specifies the private IP address bound to the backend server. The value range
   is from **1** to **64**.
-  + If `subnet_cidr_id` is left blank, IP as a Backend is enabled. In this case, the IP address must be an **IPv4** address.
-  + If `subnet_cidr_id` is not left blank, the IP address can be **IPv4** or **IPv6**. It must be in the subnet specified
-    by `subnet_cidr_id` and can only be bound to the primary NIC of the backend server.
+  + If `subnet_id` is left blank, IP as a Backend is enabled. In this case, the IP address must be an **IPv4** address.
+  + If `subnet_id` is not left blank, the IP address can be **IPv4** or **IPv6**. It must be in the subnet specified
+    by `subnet_id` and can only be bound to the primary NIC of the backend server.
 
 * `role` - (Required, String, ForceNew) Specifies the type of the backend server. The value range is from **0** to **36**.
```
Comment (Jason-Zhang9309) [inline docs/resources/elb_active_standby_pool.md]:
```suggestion
* `role` - (Required, String, ForceNew) Specifies the type of the backend server.
```

File: docs/resources/elb_active_standby_pool.md [hunk 3]
```diff
@@ -79,10 +78,9 @@ The following arguments are supported:
 
   -> **NOTE:** At least one of `loadbalancer_id`, `listener_id`, `type` must be specified.
 
-* `name` - (Optional, String, ForceNew) Specifies the backend server group name.
+* `name` - (Optional, String, ForceNew) Specifies the name of the backend server group.
```
Comment (Jason-Zhang9309) [inline docs/resources/elb_active_standby_pool.md]:
```suggestion
* `name` - (Optional, String, ForceNew) Specifies the name of the pool.
```
the resource is named pool, there should not be anything 'about backend server group'
so please check other parameters and refer to the doc of resource elb_pool to make the description better

File: docs/resources/elb_active_standby_pool.md [hunk 4]
```diff
@@ -8,29 +8,29 @@ Manages an ELB active standby pool resource within HuaweiCloud.
 
 ## Example Usage
 
-### Create a active standby Pool
+### Create an active standby Pool
```
Comment (Jason-Zhang9309) [inline docs/resources/elb_active_standby_pool.md]:
```suggestion
### Create an active-standby Pool
```

File: docs/resources/elb_active_standby_pool.md [hunk 5]
```diff
@@ -8,29 +8,29 @@ Manages an ELB active standby pool resource within HuaweiCloud.
...
-  description     = "test"
+  name            = "test_active_standby_pool"
+  description     = "test description"
   protocol        = "TCP"
   vpc_id          = var.vpc_id
   type            = "instance"
   any_port_enable = false
 
   members {
     address       = "0.0.0.1"
```
Comment (Jason-Zhang9309) [inline docs/resources/elb_active_standby_pool.md]: I think the address should be more reasonable

File: docs/resources/elb_active_standby_pool.md (hunk 1 above)
Comment (houpeng80) [reply docs/resources/elb_active_standby_pool.md]: done

File: docs/resources/elb_active_standby_pool.md (hunk 2 above)
Comment (houpeng80) [reply docs/resources/elb_active_standby_pool.md]: done

File: docs/resources/elb_active_standby_pool.md (hunk 3 above)
Comment (houpeng80) [reply docs/resources/elb_active_standby_pool.md]: done

File: docs/resources/elb_active_standby_pool.md (hunk 5 above)
Comment (houpeng80) [reply docs/resources/elb_active_standby_pool.md]: fixed

**Pull request merged**